pylintrc

```diff
@@ -179,4 +180,4 @@ int-import-graph =
 [EXCEPTIONS]
 overgeneral-exceptions = Exception
 
-# 3dd01da1adbb2cac2fbed3cbc578b3f647b792c7
```

Comment:
Sorry if I'm being dense, but I'm not sure I understand this:

>  "edx-lint write" includes a hash at the end of the file so that edits to the generated file will be detected in the future to prevent losing edits

Aren't `pylintrc` files already under version control? What's the use-case that would cause a loss of changes?